web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/ActionId/ProofDebuging/Debugger/index.tsx

```diff
@@ -0,0 +1,254 @@
+        setMessage(messages.SUCCESS_ONCHAIN);
+        // posthog.capture("debugger-verify-success", { environment: "on-chain" });
+      } else {
+        setMessage(messages.SUCCESS_PENDING);
+        // posthog.capture("debugger-verify-success", { environment: "cloud" });
+      }
+    } catch (err) {
+      setStatus(Status.ERROR);
+      setMessage(messages.REQUEST_ERROR);
+    }
+  };
+
+  const formatCode = useCallback(
+    async (data: TestProofFormValues) => {
+      try {
+        const json = JSON5.parse(data.verification_response ?? "");
+        setValue("verification_response", JSON5.stringify(json, null, 2));
+      } catch (e) {
+        setError("verification_response", {
+          type: "manual",
+          message: "Unable to auto format the response",
+        });
+      }
+    },
+    [setValue, setError]
+  );
+  return (
+    <form
+      className="w-full grid-cols-2 grid items-start justify-between gap-x-32"
+      onSubmit={handleSubmit(submit)}
+    >
+      <div className="w-full grid gap-y-6">
+        <h1 className="text-lg font-[550]">Output parameters</h1>
+        <div className="grid gap-y-8">
+          <Input
+            register={register("signal")}
+            helperText="Enter the signal as passed to IDKit"
+            label="Signal"
+            placeholder="my_signal"
+          />
+          <TextArea
+            register={register("verification_response")}
+            helperText="These are the parameters you get from the JS widget"
+            label="Verification response"
+            errors={errors?.verification_response}
+            placeholder={exampleVR}
+            className="h-[200px]"
+          />
+          <DecoratedButton
+            type="button"
+            className="w-44 text-sm"
+            onClick={() => handleSubmit(formatCode)()}
+          >
+            Format Response
+          </DecoratedButton>
+        </div>
+      </div>
+      <div className="w-full grid gap-y-6">
+        <h1 className="text-lg font-[550]">Results</h1>
+        <div
+          className={clsx(
+            "w-full h-80 bg-grey-50 border border-grey-100 flex flex-col items-center justify-center rounded-lg",
+            {
+              "bg-system-success-50 border-0": status === Status.SUCCESS,
+              "bg-system-error-50 border-0": status === Status.ERROR,
+            }
+          )}
+        >
+          <CircleIconContainer
+            variant={
+              status === Status.IDLE
+                ? "muted"
+                : status === Status.SUCCESS
+                  ? "success"
+                  : "error"
+            }
+          >
+            {status === Status.IDLE && <WorldcoinIcon />}
+            {status === Status.SUCCESS && (
+              <CheckIcon className="text-system-success-500" />
+            )}
+            {status === Status.ERROR && (
+              <CloseIcon className="text-system-error-500 stroke-3 w-5 h-5" />
```

Comment:
@andy-t-wang Coloring of the containing icon is included in `CircleIconContainer`. Let's remove colors from here, it somehow breaks coloring, and the icon is black instead.

Also stroke-3 classname not working at all as I can see.